src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,934 @@
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('40');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowLeft' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('35');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowRight' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('40');
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'Home' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('0');
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'End' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('100');
+  expect(handleOnChange).toHaveBeenCalledTimes(4);
+
+  // triggering an update with same value should not trigger callback
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'End' });
+  });
+  expect(handleOnChange).toHaveBeenCalledTimes(4);
+});
+
+it('focused thumb on disabled slider should NOT process keystrokes', () => {
+  let element: HTMLDivElement | null = null;
+  const onRef = (ref: HTMLDivElement) => {
+    element = ref;
+  };
+  const wrapper = render(
+    <Slider ref={onRef} values={[50]} step={5} disabled />,
+  );
+  const { container } = wrapper;
+  assertBaseElement(container);
+  expect(element).toBeTruthy();
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+
+  // force thumb to have focus
+  thumb.focus();
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('50');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowLeft' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('50');
+});
+
+it('thumb should show tooltip on hover', () => {
+  let element: HTMLDivElement | null = null;
+  const onRef = (ref: HTMLDivElement) => {
+    element = ref;
+  };
+  const wrapper = render(<Slider ref={onRef} values={defaultSingleValue} />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  expect(element).toBeTruthy();
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+  expect(container.querySelector('.iui-tooltip')).toBeFalsy();
+  act(() => {
+    fireEvent.mouseEnter(thumb);
+  });
+  expect(
+    (container.querySelector('.iui-tooltip') as HTMLDivElement).textContent,
+  ).toBe('50');
+  act(() => {
+    thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+    fireEvent.mouseLeave(thumb);
+  });
+});
+
+it('thumb should show tooltip on focus', () => {
```

Comment:
There is such test already...
Would be great to maybe group them together, so would be easier to see, whats already existing (i.e. all tooltip tests going one after another)